cms/djangoapps/contentstore/utils.py

```diff
@@ -180,14 +179,16 @@ def get_pages_and_resources_url(course_module):
     return pages_and_resources_url
 
 
-def get_proctored_exam_settings_url(course_module):
+def get_proctored_exam_settings_url(course_module) -> str:
     """
     Gets course authoring microfrontend URL for links to proctored exam settings page
     """
-    course_authoring_microfrontend_url = ''
+    proctored_exam_settings_url = ''
     if settings.FEATURES.get('ENABLE_EXAM_SETTINGS_HTML_VIEW'):
```

Comment:
I think its a good chance to refactor this code and start using `SettingsDictToggle`.
Move the following code in `toggles.py`

```
EXAM_SETTINGS_HTML_VIEW = SettingDictToggle(
    "FEATURES", "ENABLE_EXAM_SETTINGS_HTML_VIEW", default=False, module_name=__name__
)
```

Use the following code here.
```
if toggles.EXAM_SETTINGS_HTML_VIEW.is_enabled():
    ...
```